apps/next/app/dashboard/page.tsx

```diff
@@ -45,6 +46,53 @@ async function findCreateUserWithClerkId({
   return newUser;
 }
 
+const ensureTeamForClerkOrg = async ({
+  clerkOrgId,
+  teamName,
+  userId,
+}: {
+  clerkOrgId: string;
+  teamName: string;
+  userId: string;
+}): Promise<void> => {
+  /* Fixes an issue where if a user already has a Metal account and then uses
+  Clerk's <OrganizationSwitcher /> to create a new org, the new Clerk org will
+  exist but won't have an associated Metal team. This function will create a
+  team for the  and then add the user to it.
+  */
+  let usersPersonalTeam = await db.query.teams
```

Comment:
nit: rename this variable to just `team` since it's not the user's personal team, it's just a team associated with a clerk org

Reply:
Done.